cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/DefaultStore.java

```diff
@@ -85,10 +90,16 @@
  * Implementation of the Store that ultimately places data into MetaDataTable.
  */
 public class DefaultStore implements Store {
-  private static final Logger LOG = LoggerFactory.getLogger(DefaultStore.class);
+
   public static final String APP_META_TABLE = "app.meta";
+  public static final String WORKFLOW_STATS_TABLE = "workflow.stats";
```

Comment:
yeah can add a comment, should be added to design doc as well, 
``mds is specific for meta data, we do not want to add workflow stats related information to mds, as its not specifically meta data``